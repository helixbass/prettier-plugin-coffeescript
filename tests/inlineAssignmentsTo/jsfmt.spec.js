run_spec(__dirname, ['coffeescript'], {inlineAssignmentsTo: ['control']})
run_spec(__dirname, ['coffeescript'], {inlineAssignmentsTo: []})
