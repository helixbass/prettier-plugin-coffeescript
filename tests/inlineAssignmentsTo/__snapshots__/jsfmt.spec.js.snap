// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`control.coffee 1`] = `
x =
  if a
    b
  else
    c

x = (aaaaaaaaaaaaaaaaaaaaaaaa if bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb)

x = while a
  b

x = (aaaaaaaaaaaaaaaaaaaaaaaa until bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb)

x = loop
  c

x = for a in b
  c

x = (aaaaaaaaaaaaaaaaaaa for aaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbb)

x = for aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
  c

x: for a in b
  c

a = try
  b
catch e
  f
finally
  g
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x = if a
  b
else
  c

x =
  aaaaaaaaaaaaaaaaaaaaaaaa if bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb

x = while a
  b

x =
  aaaaaaaaaaaaaaaaaaaaaaaa until bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb

x = loop
  c

x = for a in b
  c

x =
  aaaaaaaaaaaaaaaaaaa for aaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbb

x = for aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa in (
  bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
)
  c

x: for a in b
  c

a = try
  b
catch e
  f
finally
  g

`;

exports[`control.coffee 2`] = `
x =
  if a
    b
  else
    c

x = (aaaaaaaaaaaaaaaaaaaaaaaa if bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb)

x = while a
  b

x = (aaaaaaaaaaaaaaaaaaaaaaaa until bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb)

x = loop
  c

x = for a in b
  c

x = (aaaaaaaaaaaaaaaaaaa for aaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbb)

x = for aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
  c

x: for a in b
  c

a = try
  b
catch e
  f
finally
  g
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x =
  if a
    b
  else
    c

x =
  aaaaaaaaaaaaaaaaaaaaaaaa if bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb

x =
  while a
    b

x =
  aaaaaaaaaaaaaaaaaaaaaaaa until bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb

x =
  loop
    c

x =
  for a in b
    c

x =
  aaaaaaaaaaaaaaaaaaa for aaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbb

x =
  for aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa in bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
    c

x:
  for a in b
    c

a =
  try
    b
  catch e
    f
  finally
    g

`;
